src/main/java/mate/academy/hibernate/relations/exception/DataProcessingException.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.hibernate.relations.exception;
+
+public class DataProcessingException extends RuntimeException {
+    public DataProcessingException(String message) {
+        super(message);
+    }
+
+    public DataProcessingException(String message, Throwable cause) {
+        super(message);
```

Comment:
The constructor should pass both 'message' and 'cause' to its superclass. Currently, it only passes 'message'. This should be changed to 'super(message, cause);' to ensure that the cause of the exception is preserved.